hackclub.com.yaml

```diff
@@ -1728,11 +1733,18 @@ knit:
 kr:
   ttl: 3600
   type: CNAME
-  value: cname.vercel-dns.com.
+  value: counterspell-busan.netlify.app.
+
 ko:
   ttl: 3600
   type: CNAME
   value: cname.vercel-dns.com.
+
+korea:
+  ttl: 3600
+  type: CNAME
+  value: counterspell-busan.netlify.app.
```

Comment:
This is the counterspell Busan site, not the site of your Hack Club in Korea.

Reply:
Also -- unless it's a hack club for the whole country, this should be a more specific domain to your hack club.